doc/REST-interface.md

```diff
@@ -13,6 +13,15 @@ Given a transaction hash: returns a transaction in binary, hex-encoded binary, o
 
 For full TX query capability, one must enable the transaction index via "txindex=1" command line / configuration option.
 
+####Send raw transaction
+`POST /rest/tx.<bin|hex|json>`
+
+Parameter: transaction=\<signed transaction\>
+
+Submits transaction (serialized, hex-encoded) to local node and network.
+
+Returns transaction id in binary, hex-encoded binary, or JSON formats, or an error if the transaction is invalid for any reason.
+
```

Comment:
I will ones we agreed on it :)

In the code in this version it's like in the other methods, determined by the ending, e.g.
POST 127.0.0.1:18332/rest/tx.hex